ydb/core/protos/config.proto

```diff
@@ -232,6 +232,7 @@ message TDomainsConfig {
 
     message TSecurityConfig {
         optional bool EnforceUserTokenRequirement = 1 [default = false];
+        optional bool EnforceUserTokenCheckRequirement = 7 [default = true]; // Check token if it was specified // If false and token was incorrect or access was denied, the request will be treated as if there was no token
```

Comment:
// Check if a token was specified
// If not, or if the token was incorrect or access was denied, the request will be handled as if no token was provided